servers/zms/src/main/java/com/yahoo/athenz/zms/DBService.java

```diff
@@ -1884,8 +1906,13 @@ void executeDeleteMembership(ResourceContext ctx, String domainName, String role
 
                 // audit log the request
 
-                auditLogRequest(ctx, domainName, auditRef, caller, ZMSConsts.HTTP_DELETE,
-                        roleName, "{\"member\": \"" + normalizedMember + "\"}");
+                if (pending) {
```

Comment:
let's simplify the call and just use `... caller, pending ? ZMSConsts.HTTP_PUT : MSConsts.HTTP_DELETE, roleName ....`